docs/sample/election-manifest.json

```diff
@@ -0,0 +1,1867 @@
+                },
+                {
+                    "objectId": "exeter-utility-district-referendum-selection",
+                    "sequenceOrder": 1,
+                    "candidateId": "exeter-utility-district-referendum"
+                }
+            ],
+            "ballotTitle": {
+                "text": [
+                    {
+                        "value": "Levy Lift to Maintain Public Safety and Other Core Utility Services",
+                        "language": "en"
+                    },
+                    {
+                        "value": "Levy Lift para Mantener la Seguridad Pública y Otros Servicios Básicos",
+                        "language": "es"
+                    }
+                ]
+            },
+            "ballotSubtitle": {
+                "text": [
+                    {
+                        "value": "Should this Proposition be approved?",
+                        "language": "en"
+                    },
+                    {
+                        "value": "Uno",
+                        "language": "es"
+                    }
+                ]
+            },
+            "ballotDescription": {
+                "text": [
+                    {
+                        "value": "The Executive Board of the Exeter Utility district adopted Ordinance No. 970 concerning voter approval of its regular property tax levy. This proposition would maintain police, fire, park and mandated community services by increasing the regular property tax levy rate above the limit factor by $0.20/$1,000 assessed value to a maximum rate of $0.83712/$1,000 assessed valuation for collection in 2020, set a 5% limit factor for each year 2021-2025, use the 2025 levy amount as the base to compute subsequent levy limits, and exempt low income seniors and disabled; as set forth in Ordinance No. 970.",
+                        "language": "en"
+                    },
+                    {
+                        "value": "La Junta Ejecutiva del distrito de Servicios Públicos de Exeter adoptó la Ordenanza No. 970 sobre la aprobación de los votantes de su recaudación de impuestos de propiedad regulatoria. Esta propuesta mantendría a la policía, los bomberos, los parques y los servicios comunitarios obligatorios al aumentar la tasa de recaudación del impuesto a la propiedad regular de por encima del factor límite en un valor tasado de $ 0.20 / $ 1,000 a una tasa máxima de $ 0.83712 / $ 1,000 de tasación tasada para cobrar en 2020, establecer un 5% factor límite para cada año 2021-2025, use el monto del impuesto 2025 como base para calcular los límites de impuestos posteriores, y exima a las personas de la tercera edad de bajos ingresos y discapacitadas; como se establece en la Ordenanza No. 970.",
+                        "language": "es"
+                    }
+                ]
+            }
+        }
+    ],
+    "ballotStyles": [
+        {
+            "objectId": "us-president",
```

Comment:
i dont think this is quite right.  the "us-president" contest is associated the the `electoralDistrictId` of `hamilton-county` which appears to the be the top level election scope, implying that the county is the one conducting the election.

first, `hamilton-county` is not included in this list of GP unit id's, so it would not be a valid contest for this ballot style.

second, the way this is set up, this "ballot style" of "us-president" would actually allow selections from all of the contests associated with the GP units in this list. 

instead i think you want to do something like this:

```
"ballotStyles": [
        {
            "objectId": "pismo-beach-school-district-precinct-1",
            "geopoliticalUnitIds": [ "hamilton-county", "lacrox-exeter-utility-district", "pismo-beach-school-district-precinct-1"]
        }
]
```

☝️  what that says is that a ballot style is the most localized scope that defines all of the contests that should appear in front of a voter.  i'm not sure if my example maps to your town layout, but in my example i'm basically saying "people in pismo beach district vote for school board, the exeter utility district, and all contests at the county level (governor and president)"

make sense?

Reply:
not at all. in my example, which I consulted with actual county clerks on, people in the Exeter Utility District WOULD NOT vote for Pismo Beach School Board (why would they? they're different locales)